ethsigner/signer/multiplatform/src/test/resources/metadata-toml-configs/unknown_type_signer_fe3b557e8fb62b89f4916b721be55ceb828dbd75.toml

```diff
@@ -0,0 +1,8 @@
+[metadata]
```

Comment:
why the crazy filename? (i.e. hex identifier?)

Reply:
I followed what was done in KeyPasswordFile - allow any prefix as long as the filename ends with the address and .toml extension